torchvision/transforms/functional_pil.py

```diff
@@ -20,6 +20,15 @@ def _is_pil_image(img: Any) -> bool:
         return isinstance(img, Image.Image)
 
 
+@torch.jit.unused
+def get_dimensions(img: Any) -> List[int]:
```

Comment:
"Low-level" kernel for pil.

We should start discouraging the use of `get_image_size()` and `get_image_num_channels()` on main TorchVision. I haven't done this yet to avoid introducing noise in the PR but I suggest doing this as last minute clean up prior merging.